### DISCUSSION TITLE
When I use lazy loading to import components, I encountered chunkLoadError：chunk loading xxx failed

### DISCUSSION THREAD

**Discussion by lsustc** [webpack/webpack#14281] [Q&A]
When I use lazy loading to import components, I encountered chunkLoadError. There are three types errors I meet: 
1、js run error 
eg.ChunkLoadError: Loading chunk SimpleMainIndex failed.\n(error: https://s.thsi.cn/cd/website-basic-base-container/dist/js/SimpleMainIndex.a6c169c4.js) 

2、request timeout 
eg.ChunkLoadError: Loading chunk SimpleMainIndex failed.\n(timeout: https://s.thsi.cn/cd/website-basic-base-container/dist/js/SimpleMainIndex.a6c169c4.js) 

3、js missing 
eg.ChunkLoadError: Loading chunk SimpleMainIndex failed.\n(missing: https://s.thsi.cn/cd/website-basic-base-container/dist/js/SimpleMainIndex.a6c169c4.js)

Our page visit is probably at the million level,the chunkloaderror ratio is about 1%.

import method: const comp = () => import(/* webpackChunkName: "SimpleMainIndex" */ '~/components/mainIndex/index.vue'); const res = await comp().default;

my project webpack version:4.46.0

**Comment by alexander-akait** [comment]
Please consider to update webpack v4 to v5, a lot of bugs were fixed, anyway please provide steps to reproduce so we can help you
